spec/controllers/users_controller_spec.rb

```diff
@@ -17,50 +17,85 @@
   end
   
   describe "POST create" do
-    context "valid input details" do
+    context "valid personal details" do
       
-      let(:stripe_helper) { StripeMock.create_test_helper }
-      before do
-        StripeMock.start
-        post :create, user: generate_attributes_for(:user), stripeToken: stripe_helper.generate_card_token 
-      end
-      after do
-        StripeMock.stop 
-        ActionMailer::Base.deliveries.clear
-      end
-        
-      it "creates user record" do
-        expect(User.count).to eq(1)
-      end
+      context "valid card details" do
+      
+        let(:attempt_card_payment) { double(:attempt_card_payment) }
+        #let(:stripe_helper) { StripeMock.create_test_helper }
+        before do
+          #StripeMock.start
+          attempt_card_payment.should_receive(:processed).and_return(true)
```

Comment:
You should use the `expect` syntax instead of the old `should` syntax:

```ruby
expect(attempt_card_payment).to receive(:processed).and_return(true)
```